lua/mp_menu/common.lua

```diff
@@ -329,7 +332,7 @@ derma.DefineControl( "MP.SidebarToggleButton", "", SIDEBAR_TOGGLE_BTN, "MP.Sideb
 
 local FAVORITE_BTN = {}
 
-AccessorFunc( FAVORITE_BTN, "Favorited", "Favorited" )
+AccessorFunc( FAVORITE_BTN, L "Favorited", "Favorited" )
```

Comment:
The *AccessorFunc* function registers access functions to the table, all you are doing is changing the internal key it will store the favorited value with interally in the table